spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb

```diff
@@ -53,15 +53,15 @@
         it "2 refreshes, first creates all entities, second deletes all entitites" do
           @data_scaling = 1
           2.times do
-            refresh_spec
+            refresh_spec(false)
```

Comment:
in L64 I introduce additional parameter that controls whether tests for specific entities are needed. Since this test deletes all entities, there is none to actually make a specific test in this case. All other cases should pass specific tests as well.